arches/app/media/js/views/graph/graph-designer/graph-tree.js

```diff
@@ -248,6 +248,20 @@ define([
         },
         toggleGrid: function(){
             this.showGrid(!this.showGrid());
+        },
+
+        copyNodeId: function(node, e) {
+            function restoreTitle(e) {
+                $(e.target).tooltip('hide') 
+                $(e.target).attr('data-original-title', 'Copy UUID to Clipboard')
```

Comment:
'Copy UUID to Clipboard' and 'Copied to Clipboard' need to be added to arches.translations. Also, 'Copy UUID to Clipboard' should probably be changed to 'Copy Node ID to Clipboard' just to be explicit.